packages/server/hubspot/hubSpotApi.ts

```diff
@@ -0,0 +1,143 @@
+import fetch from 'node-fetch'
+const accessToken = process.env.HUBSPOT_API_KEY
+
+type Company = {
+  id: string
+  archived: boolean
+}
+
+type DealsResponse = {
+  total: number
+}
+
+type CompaniesResponse = {
+  results: Company[]
+}
+
+const getCompaniesByDomain = async (domain: string): Promise<CompaniesResponse | Error> => {
```

Comment:
+1 it looks like the HubSpot Client package offers types, so we wouldn't need to cast them: https://www.npmjs.com/package/@hubspot/api-client. It's probably not worth refactoring though unless we expect to add more logic to the hubspot API soon

Reply:
Make sense, It's completely slipped my mind that it's possible to use a library here.
I will add a comment in the code, so we can take this library next time.